packages/zowe-explorer/__tests__/__unit__/utils/AuthUtils.unit.test.ts

```diff
@@ -104,6 +104,42 @@ describe("AuthUtils", () => {
             isUsingTokenAuthMock.mockRestore();
         });
     });
+
+    describe("isUsingTokenAuth", () => {
+        it("should return false if shouldRemoveTokenFromProfile() returns true", async () => {
+            const profile = { name: "aProfile", type: "zosmf" } as any;
+            new MockedProperty(Constants, "PROFILES_CACHE", {
+                value: {
+                    ssoLogin: jest.fn().mockImplementation(),
+                    promptCredentials: jest.fn().mockImplementation(),
+                    getDefaultProfile: jest.fn().mockReturnValue("sestest"),
+                    shouldRemoveTokenFromProfile: jest.fn().mockReturnValue(true),
+                    loadNamedProfile: jest.fn(),
+                } as any,
+                configurable: true,
+            });
+
+            const usingTokenAuth = await AuthUtils.isUsingTokenAuth(profile.name);
+            expect(usingTokenAuth).toBe(false);
+        });
+        it("should return true if getPropsForProfile() returns tokenValue", async () => {
+            const profile = { name: "aProfile", type: "zosmf" } as any;
+            new MockedProperty(Constants, "PROFILES_CACHE", {
+                value: {
+                    ssoLogin: jest.fn().mockImplementation(),
+                    promptCredentials: jest.fn().mockImplementation(),
+                    getDefaultProfile: jest.fn().mockReturnValue("sestest"),
+                    shouldRemoveTokenFromProfile: jest.fn().mockReturnValue(false),
+                    loadNamedProfile: jest.fn(),
+                    getPropsForProfile: jest.fn().mockReturnValue(["tokenValue"]),
+                } as any,
+                configurable: true,
+            });
+
+            const usingTokenAuth = await AuthUtils.isUsingTokenAuth(profile.name);
+            expect(usingTokenAuth).toBe(true);
```

Comment:
Can you save the `MockedProperty` instance to a variable (e.g. `profilesCacheMock`) and then add this line to the bottom of both of these test cases?

```ts
profilesCacheMock[Symbol.dispose]();
```

This ensures that the mocked property is cleaned up between tests and avoids impacting other tests with code that may interact with `Constants.PROFILES_CACHE`.